nipy/algorithms/clustering/gmm.py

```diff
@@ -217,8 +218,8 @@ def plot2D(x, my_gmm, z=None, with_dots=True, log_scale=False, mpaxes=None,
         if z is None:
             plt.plot(x[:, 0], x[:, 1], 'o')
         else:
-            hsv = plt.cm.hsv(range(256))
-            col = hsv[range(0, 256, 256 // int(z.max() + 1))]
+            hsv = plt.cm.hsv(list(range(256)))
+            col = hsv[list(range(0, 256, 256 // int(z.max() + 1)))]
```

Comment:
I guessed that was true, but I left these in because we otherwise depend on the fact that `plt.cm.hsv` accepts an iterator as well as a sequence, which I suppose is not guaranteed, but just happens to be the case (nothing I could see in the docstring to help).